### PR TITLE
added operations modules

### PR DESCRIPTION
**Pull request by Harshvardhan80**
Targets #2783 

Description
Checklist:
 I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/Operations/Resources/views/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@extends('operations::layouts.master')
```
Comment (GauravGusain98) [inline Modules/Operations/Resources/views/index.blade.php:1]: You should move this file inside Module/Operations/Resources/views/office-location and create index file inside that. Then in controller as well you should call that file.

File: Modules/Operations/Routes/web.php [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+|--------------------------------------------------------------------------
+|
+| Here is where you can register web routes for your application. These
+| routes are loaded by the RouteServiceProvider within a group which
+| contains the "web" middleware group. Now create something great!
+|
+*/
+
+Route::prefix('operations')->group(function () {
+    Route::get('/', 'operationsController@index')->name('office_location.index');
```
Comment (GauravGusain98) [inline Modules/Operations/Routes/web.php:15]: the url should be `operation/office-location`

File: resources/views/layouts/navbar.blade.php [hunk 3]
```diff
@@ -113,6 +113,17 @@
             </div>
         </li>
     @endcan
+    @can('operations.view')
```
Comment (GauravGusain98) [inline resources/views/layouts/navbar.blade.php:116]: Where is this permission created. You need to create this permission and assign this permission to `admin` and `super-admin`

Comment (GauravGusain98) [reply resources/views/layouts/navbar.blade.php:116]: Check how we have created other permissions and assign them to different role.

Comment (Harshvardhan80) [reply resources/views/layouts/navbar.blade.php:116]: i have done

File: Modules/Operations/Resources/views/index.blade.php (hunk 1 above)
Comment (Harshvardhan80) [reply Modules/Operations/Resources/views/index.blade.php]: i have done

File: Modules/Operations/Routes/web.php [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+|--------------------------------------------------------------------------
+|
+| Here is where you can register web routes for your application. These
+| routes are loaded by the RouteServiceProvider within a group which
+| contains the "web" middleware group. Now create something great!
+|
+*/
+
+Route::prefix('operations')->group(function () {
+    Route::get('/operations/office_location', 'operationsController@index')->name('office-location.index');
```
Comment (GauravGusain98) [inline Modules/Operations/Routes/web.php:15]: As per your implementation, the URL for this route will become `operations/operations/office_location` but it should be `operations/office_location`

File: composer.lock
Comment (GauravGusain98) [inline composer.lock:1]: We don't need any changes in this file. So please revert the changes done to this file

File: package-lock.json
Comment (GauravGusain98) [inline package-lock.json:1]: We don't need any changes in this file. So please revert the changes done to this file

File: Modules/Operations/Database/Seeders/OperationsDatabaseSeeder.php [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        Model::unguard();
+
```
Comment (GauravGusain98) [inline Modules/Operations/Database/Seeders/OperationsDatabaseSeeder.php:18]: I think here you need to call the `OperationsPermissionTableSeeder`. Check other modules like Project, HR etc.

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
minor suggestions please check

**Pull request closed without merging**